source/beamlines.md

```diff
@@ -79,22 +79,26 @@ placement       # Structure. Shifts element or subline longitudinally.
 
 Example with four `items` in `line`:
 ```{code} yaml
+Sextupole:
+  name: thingB
+
 BeamLine:
   name: inj_line
   multipass: True
   length: 37.8
   zero_point: thingC
-  - kind: Line
-    name: line
-    - thingB            # This item refers to the name of an element or BeamLine defined elsewhere.
-    - kind: Drift:      # Another way of specifying the name of an element or BeamLine as an item.
-        name: thingC
-    - kind: Quadrupole: # This item contains a Quadrupole that is reversed.
+  line:
+    - thingB               # This item refers to the name of an element or BeamLine defined elsewhere.
+    - lineitem:
+        inherit: thingB    # Another way of referring to thingC
```

Comment:
How is this "referring to thingC"? Do you mean "thingB" or am I missing something?